### PR TITLE
Core service restructure

### PR DESCRIPTION
**Pull request by Ethanljf**
These changes address concerns raised in issue https://github.com/canada-ca/tracker/issues/2453

This removes the core service and splits the operations into two services. "Guidance" will update guidance and criteria and will run before scanning, "summaries" will run after scanning to ensure that summaries are calculated correctly.

### CODE REVIEW

File: app/bases/knative/config/summaries.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: summaries
+  namespace: scanners
+spec:
+  schedule: "0 5 * * *"
```
Comment (nsdeschenes) [inline app/bases/knative/config/summaries.yaml:7]: How can we guarantee that this is always after the scans have been ran?

File: services/summaries/summaries.py [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+def update_scan_summaries(host=DB_HOST, name=DB_NAME, user=DB_USER, password=DB_PASS, port=DB_PORT):
+    logging.info(f"Updating scan summaries...")
+
+    # Establish DB connection
+    connection_string = f"http://{host}:{port}"
+    client = ArangoClient(hosts=connection_string)
+    db = client.db(name, username=user, password=password)
+
+    if not db.has_collection("scanSummaries"):
+        db.create_collection(
```
Comment (nsdeschenes) [inline services/summaries/summaries.py:34]: This needs to be removed for the database initilization script to be the so it does not override any settings that we could possibly change.

File: services/summaries/summaries.py [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+def update_chart_summaries(host=DB_HOST, name=DB_NAME, user=DB_USER, password=DB_PASS, port=DB_PORT):
+    logging.info(f"Updating chart summaries...")
+
+    # Establish DB connection
+    connection_string = f"http://{host}:{port}"
+    client = ArangoClient(hosts=connection_string)
+    db = client.db(name, username=user, password=password)
+
+    if not db.has_collection("chartSummaries"):
+        db.create_collection(
```
Comment (nsdeschenes) [inline services/summaries/summaries.py:85]: This needs to be removed for the database initilization script to be the so it does not override any settings that we could possibly change.

**Review by nsdeschenes — CHANGES_REQUESTED** [review]
The `update_guidance` function also needs the database creation functions removed, same reasons as above.

**Review by ThomasNickerson — CHANGES_REQUESTED** [review]
We also need to update the flux image policy and flux image registry to reflect the split, probably splitting it up the same way everything else was split makes the most sense.

Everything else looks great!

https://github.com/canada-ca/tracker/blob/master/deploy/bases/core-registry.yaml
https://github.com/canada-ca/tracker/blob/master/deploy/gke/core-image-policy.yaml

File: app/bases/knative/config/summaries.yaml (hunk 1 above)
Comment (Ethanljf) [reply app/bases/knative/config/summaries.yaml]: Updated to reflect the schedule agreed upon in our discussion

**Review by ThomasNickerson — APPROVED** [review]

**Pull request merged**